Update PPSSPP developer setup documentation

pandakar-gaming opened the pull request:
I noticed that PPSSPP released a new public version the other day, and with that they have support for RAIntegration as part of the public build! This PR updates a bit of the developer documentation on our side to match what someone needs to do today to get PPSSPP setup properly for development.

The DLL rename is intentional. I noticed after helping someone that their error message was a bit misleading (asked them to rename one way when the emulator looks for another way) and asked them to update it here https://github.com/hrydgard/ppsspp/issues/19591 .

pandakar-gaming committed:
Update emulator-setup-for-developers.md

Updating documentation for PPSSPP setup as the latest public release now supports RAIntegration for developers. It also seems to have a DLL rename.

pandakar-gaming committed:
Update ppsspp-raintegration-folder.png

This changes the visual filename that the reader sees when attempting to set up PPSSPP with RA_Integration.dll so that it matches the written description provided.

wescopeland commented on docs/developer-docs/emulator-setup-for-developers.md:
```diff
@@ -75,22 +75,30 @@ When an update for RAIntegration is available, it will ask you if you want to up
 ## PPSSPP
 
-PPSSPP currently requires a development build for RAIntegration support:
+PPSSPP supports RAIntegration in its main builds now as of version 1.18. You can grab the latest version here:
```
```suggestion
PPSSPP supports RAIntegration in its main builds as of version 1.18. You can grab the latest version here:
```

pandakar-gaming replied:
done

wescopeland commented on docs/developer-docs/emulator-setup-for-developers.md:
```diff
@@ -75,22 +75,30 @@ When an update for RAIntegration is available, it will ask you if you want to up
 
+### Portable Install
+
+The installer from here downloads PPSSPP as an installer. If you want to avoid installing PPSSPP and run it as a portable application in its own folder, you can grab one of the development builds from here:
```
This sentence reads kinda weird. Wondering if we can tighten this up a bit. Something to the effect of:

> To download PPSSPP as a portable application, bypassing the need for installation, you can grab one of the development builds from here. Otherwise, you can download the installer version from this link:

pandakar-gaming replied:
I think I can do slightly better than that and pull this into the section above, on a reread I'm not sure this needs to be its own section. Thinking from a loose structure:

1. Tell the reader where to get the emulator
2. Tell the reader how to set up the emulator via the emulator itself
3. If the reader chose a manual install, give the reader instructions on how to ensure the manual install functions.

wescopeland commented on docs/developer-docs/emulator-setup-for-developers.md:
```diff
@@ -75,22 +75,30 @@ When an update for RAIntegration is available, it will ask you if you want to up
+
+![RAIntegration in PPSSPP folder](/public/ppsspp-raintegration-folder.png)
+
+After that, follow the instructions from above to Enable RAIntegration (for achievement development) and then reboot PPSSPP. You should get a success confirmation.
```
```suggestion
After that, follow the instructions from above to Enable RAIntegration (for achievement development) and then restart PPSSPP. You should get a success confirmation.
```

pandakar-gaming replied:
The overall wording changed a bit after pulling this into the section above, but I did make sure to use the term "restart" over "reboot"

wescopeland reviewed (commented):
Thanks for doing this! Just a few minor nit pieces of feedback.

pandakar-gaming committed:
Update descriptions for clarity

Also removes section for portable install so that the whole thing flows a bit better.
